src/sql/src/parsers/create_parser.rs

```diff
@@ -324,7 +325,22 @@ impl<'a> ParserContext<'a> {
             .expect_keyword(Keyword::AS)
             .context(SyntaxSnafu)?;
 
-        let query = self.parser.parse_query().context(error::SyntaxSnafu)?;
+        let start_loc = self.parser.peek_token().span.start;
```

Comment:
I don't get  the refactor of the parser. Could you explain in details?

Reply:
> I don't get the refactor of the parser. Could you explain in details?

this is for fixing the problem that prom ql parser will parse `now()` to a fixed timestamp, so instead just slice the entire raw query string away and save it